### PR TITLE
renamed from pytui to wtpython

### PR DESCRIPTION
**Pull request by briancohan**

### CODE REVIEW

**Review by mustafaquraish — APPROVED** [review]

**Comment by mustafaquraish** [comment]
feels weird... I liked `pytui` :( But it is more ergonomic

**Review by ethansocal — APPROVED** [review]

**Comment by VagishVela** [comment]
I haven't tested this but it looks good. Did @ethansocal test it? If so LGTM!

**Comment by briancohan** [comment]
> I haven't tested this but it looks good. Did @ethansocal test it? If so LGTM!

Ethan and mustafa have both "approved" i was waiting for your "approval" before merging

**Pull request merged**
